app/models/version.rb

```diff
@@ -0,0 +1,13 @@
+class Version < ApplicationRecord
+  belongs_to :assessment
+  belongs_to :format
+  belongs_to :created_by, class_name: 'UserProfile'
```

Comment:
Might be overreacting to too many "profiles" in Tutor, but could we just make `UserProfile` into `User`?

Reply:
yes!  I almost did that actually 😁 but then figured I should follow Tutor's lead